### PR TITLE
initial version for latex

### PR DESCRIPTION
**Pull request by icymist**
Initial version for LaTeX.

First time requesting for a pull. Let me know if something is not right.

### CODE REVIEW

File: latex.html.markdown [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+about the formatting. It is also great if one wants to create documents
+containing a lot of mathematics. Getting a good document is very easy, but
+getting it to behave exactly the way you want can be a bit hairy.
+
+
+```latex
+% All comment lines start with %
+% There are no multi-line comments
+
+% LaTeX is NOT a ``What You See Is What You Get'' word processing software like
```
Comment (levibostian) [inline latex.html.markdown]: You are using a combination of `` and '' here. Do 'What You See Is What You Get'

File: latex.html.markdown [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+% MS Word, or OpenOffice Writer
```
Comment (levibostian) [inline latex.html.markdown]: If Latex is not this, what is it?

File: latex.html.markdown [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+% You write all your formatting information in plain text in Step 1.
+% The compilation part in Step 2 takes care of producing the document in the
+% format you defined in Step 1.
+
+% For Step 1, it is best if you get a good text editor
+% On Windows, probably Notepad++
+% For Step 2, you will need to get a TeX distribution
+% Windows: MikTeX
+% MacOS: MacTeX
+% Linux: Should be available from your package manager
```
Comment (levibostian) [inline latex.html.markdown]: What is the package called?

**Comment by levibostian** [comment]
The document flow is kind of confusing. This seems more like a blog post tutorial telling you how to get LaTeX up and running. LearnXinYminutes is usually used for a quick learning tool or reference. 

For example, the part where you talk about the document{begin} and document{end}, there is a lot of repeating there. Just state document{begin} once and then state what that means. Make it flow a little better. 

Make sense?

**Comment by geoffliu** [comment]
Building on what @levibostian is saying, I think it would be nice to break this doc into sections using the `\section`s, rather than with comments, as we do with other docs.

**Pull request merged**